libnmstate/nm/ipv6.py

```diff
@@ -46,15 +52,22 @@ def get_info(active_connection):
 
 def create_setting(config):
     setting_ip = nmclient.NM.SettingIP6Config.new()
+    setting_ip.props.method = nmclient.NM.SETTING_IP6_CONFIG_METHOD_IGNORE
     if config and config.get('enabled') and config.get('address'):
         setting_ip.props.method = (
             nmclient.NM.SETTING_IP6_CONFIG_METHOD_MANUAL)
         for address in config['address']:
+            addr_str = '{a[ip]}/{a[prefix-length]}'.format(a=address)
+            if is_link_local_addr(addr_str):
+                logging.warning('IPv6 link local address {} is ignored when '
+                                'applying desired state'.format(addr_str))
+                continue
```

Comment:
I would prefer you use `else`, if possible, avoid `break` and `continue` in mid loop, it is bug prone (not always possible).

Reply:
Done.